.github/workflows/ci.yml

```diff
@@ -185,13 +184,13 @@ jobs:
       - name: Launch external services
         if: ${{ success() || failure() }}
         run: |
-          docker-compose --file packages/hash/external-services/docker-compose.yml build
-          docker-compose --file packages/hash/external-services/docker-compose.yml up --detach
+          yarn external-services up --build --detach
```

Comment:
I investigated how we can replace `--detach` with `--wait` to ensure everything is healthy before the command exits. In theory, this is possible but needs some adjustments to the docker-compose.yml structure. I will address this in a separate PR to not block this.

Reply:
Thanks! Have you created a task for this?

Reply:
I added [this Asana task](https://app.asana.com/0/1200211978612931/1203122830714072). I did some work on this in [`wait-on-docker`](https://github.com/hashintel/hash/tree/td/wait-on-docker), and will continue working in that as soon as `dev/graph` is merged into `main`